csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -1022,46 +1022,6 @@ module Private {
     }
   }
 
-  /** Provides a query predicate for outputting a set of relevant flow summaries. */
```

Comment:
I still think it makes sense to have here, in case other languages want to use it later.